includes/class-sensei-grading.php

```diff
@@ -211,10 +211,28 @@ public function grading_default_view() {
 		$sensei_grading_overview->prepare_items();
 
 		// Wrappers
+		/**
+		 * Fires before the container of the Grading page.
+		 *
+		 * @hook grading_before_container
+		 */
 		do_action( 'grading_before_container' );
+		/**
+		 * Fires before the container of the Grading page.
+		 *
+		 * @hook grading_before_container
```

Comment:
```suggestion
		 * @hook grading_wrapper_container
```

Is the description up-to-date?